### ISSUE TITLE
PA REVIEW: List summary

### ISSUE THREAD

**Issue by walterhwan** [walterhwan/remember_the_bacon#7]
- [ ] showing info on estimated time, num of tasks, num of completed
- [ ] Adequate styling
- [ ] Smooth, bug-free navigation
- [ ] Adequate and appropriate seeds to demonstrate the feature

**Comment by AAlfarho** [comment]
As the `tasks` feature is tightly involved with the list summary,  Issue #5 has feedback regarding this MVP, the following is an extract of such issue:

- [x] Changing a task's estimate or to `completed`, is not updating the overall `List` information, only after a refresh, this information is in sync.
- [x] Currently, the only way to update the _value_ of tasks it's by clicking it, let's add something like _rememberthemilk_ to let users know they can click the task name to update its value. 
    ![screen shot 2018-04-11 at 9 43 09 am](https://user-images.githubusercontent.com/1103663/38630802-c928ced8-3d6c-11e8-9abe-2d28b6eb486f.png)
- [x] The only way to persist a new value for a task its by hitting `enter` after updating the value. This means that changes will not be saved if users click on a different task or click the _close button_ without hitting `enter`, this is not intuitive, we should  **autosave** the new task's value on the spot, we can do this several ways.
   + Have a debounced function that saves the task value on input change, **NOTE**, this does not mean trigger a new dispatch and hit our server every time the user updates the input, we should throttle or debounce this functionality, take a look at _lodash_ implementation (https://css-tricks.com/debouncing-throttling-explained-examples/).
  + On componentWillReceiveProps or on the close action, we can trigger an update, I personally like the previous option better, given that your other components (Tasks Index) will update live.

Feel free to mark the checkboxes yourself after addressing them. Assign this issue back to me once the updates are live in Heroku.

**Comment by AAlfarho** [comment]
Closing as this issues were already addressed in #6.

MVP is now complete.
